wpilibj/src/main/java/edu/wpi/first/wpilibj/InterruptHandler.java

```diff
@@ -0,0 +1,22 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
```

Comment:
Just want to make sure the date range is accurate.

Reply:
This is going to happen with a lot of existing PR's that added new files, because the date change happened while they were in development. I don't think the development effort is worth going through every PR to fix this.